src/makeCalculator.js

```diff
@@ -3,8 +3,29 @@
 /**
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
-}
 
 module.exports = makeCalculator;
+
+function makeCalculator() {
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    add: (num) => (result += num),
+    subtract: (num) => (result -= num),
+    multiply: (num) => (result *= num),
+    divide: (num) => (result /= num),
+    reset: function () {
+      result = 0;
+
+      return this;
+    },
+    operate: function (callback, num) {
```

Comment:
[CODE KNOWLEDGE] - When creating a method in an object, you don't need to use the `function` keyword. Use the shortcut instead. For example, `operate(callback, num) { ... }`.